frontend/src/pages/PipelineDetails.test.tsx

```diff
@@ -244,8 +265,8 @@ describe('PipelineDetails', () => {
               ),
             },
             {
-              displayName: testRun.run!.name,
```

Comment:
We might want to preserve V1 test because we still support V1 pipeline. Can you make a copy of this test for v1?

Reply:
For build breadcrumb part, we only use v2 getRun() / getRecurringRun() request, so I think we only need v2 test.

Reply:
If the parameters required in some logics are retrievable from both v1 and v2 API response, we then select v2.